src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -484,18 +484,45 @@
 	display: none;
 }
 
+.interactive-session .chat-used-context-list {
+	border: 1px solid var(--vscode-chat-requestBorder);
+	border-radius: 3px;
+	padding: 4px;
+}
+
+.interactive-session .chat-used-context-list .monaco-list .monaco-list-row {
+	border-radius: 2px;
+}
+
 .interactive-session .chat-used-context-label {
-	font-size: 0.9em;
+	font-size: 12px;
+	color: var(--vscode-foreground);
+	opacity: 0.8;
+}
+
+.interactive-session .chat-used-context-label:hover {
+	opacity: unset;
 }
 
 .interactive-session .chat-used-context-label .monaco-button {
 	/* unset Button styles */
 	display: inline-flex;
-	width: initial;
+	width: 100%;
 	border: none;
 	padding: 0;
 	text-align: initial;
-	padding-left: 4px;
 	justify-content: initial;
-	margin-bottom: 3px;
+	margin-bottom: 6px;
+}
+
+.interactive-session .chat-used-context-label .monaco-text-button {
+	outline-offset: unset !important;
+}
+
+.interactive-session .chat-used-context-label .monaco-button:focus-within {
+	outline: none;
```

Comment:
I think we'll need to keep this, otherwise you can't tell where focus is when you tab to this button

Reply:
Weird, I thought I put this back after filing an issue about only showing this on keyboard focus vs. click (the current behavior). I'll get this back in.

Reply:
https://github.com/microsoft/vscode/pull/195583